src/main/java/commands/diet/dietmanager/DietSessionClear.java

```diff
@@ -41,7 +41,7 @@ public void execute(String input, DietStorage storage) {
 
     private boolean checkConfirmation() {
```

Comment:
This Confirmation method is duplicated in multiple places, so maybe we can extract them out into utils?

Reply:
@zsk612 @CFZeon

Reply:
Actually, it's used in workout manager and workout session as well. So can merge this PR first then I'll create a util method for that and you guys refactor your code accordingly